content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -5,30 +5,40 @@ weight: 65
 keywords: [security,health-check]
 ---
 
-This task shows how to use [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) for health checking of Istio services.
+This task shows how to use [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
+for health checking of Istio services.
```

Comment:
to check the health of the Istio services.

Reply:
done